RHEL/7/input/xccdf/system/accounts/banners.xml

```diff
@@ -148,7 +148,7 @@ process and facilitates possible legal action against attackers.
 </rationale>
 <ident cce="26892-0" />
 <oval id="dconf_gnome_login_banner_text" value="login_banner_text"/>
-<ref nist="AC-8(a),AC-8(b),AC-8(c)" disa="48,50,1384,1385,1386,1387,1388" ossrg="23" />
+<ref nist="AC-8(a),AC-8(b),AC-8(c)" disa="48" ossrg="SRG-OS-000023-GPOS-00006, SRG-OS-000024-GPOS-00007, SRG-OS-000228-GPOS-00088" stigid="010031" />
```

Comment:
On 7/28/16 9:22 PM, redhatrises wrote:
>
> In RHEL/7/input/xccdf/system/accounts/banners.xml 
> <https://github.com/OpenSCAP/scap-security-guide/pull/1358#discussion_r72730487>:
>
> > @@ -148,7 +148,7 @@ process and facilitates possible legal action against attackers.
> >  </rationale>
> >  <ident cce="26892-0" />
> >  <oval id="dconf_gnome_login_banner_text" value="login_banner_text"/>
> > -<ref nist="AC-8(a),AC-8(b),AC-8(c)" disa="48,50,1384,1385,1386,1387,1388" ossrg="23" />
> > +<ref nist="AC-8(a),AC-8(b),AC-8(c)" disa="48" ossrg="SRG-OS-000023-GPOS-00006, SRG-OS-000024-GPOS-00007, SRG-OS-000228-GPOS-00088" stigid="010031" />
>
> @shawndwells <https://github.com/shawndwells> slight nitpick. Can you 
> remove spaces between the SRG identifiers so that we are consistent 
> with the rest of the SRG identifiers?
>
Good catch. Updated!